static/app/views/monitors/components/monitorQuickStartGuide.tsx

```diff
@@ -0,0 +1,93 @@
+import {useState} from 'react';
+import styled from '@emotion/styled';
+
+import {CompactSelect} from 'sentry/components/compactSelect';
+import {PlatformKey} from 'sentry/data/platformCategories';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {ProjectKey} from 'sentry/types';
+import {useQuery} from 'sentry/utils/queryClient';
+import {
+  CLICronQuickStart,
+  CurlCronQuickStart,
+  PythonCronQuickStart,
+  QuickStartProps,
+} from 'sentry/views/monitors/components/quickStartEntries';
+
+import {Monitor} from '../types';
+
+interface Props {
+  monitor: Monitor;
+  orgId: string;
+}
+
+interface OnboardingGuide {
+  Guide: React.ComponentType<QuickStartProps>;
+  label: string;
+  platforms?: Set<PlatformKey>;
+}
+
+const onboardingGuides: Record<string, OnboardingGuide> = {
+  cli: {
+    label: 'Sentry CLI',
+    Guide: CLICronQuickStart,
+  },
+  curl: {
+    label: 'cURL',
+    Guide: CurlCronQuickStart,
+  },
+  python: {
+    label: 'Python',
+    Guide: PythonCronQuickStart,
+    platforms: new Set(['python', 'python-celery']),
+  },
+};
+
+const guideToSelectOption = ({key, label}) => ({label, value: key});
+
+export default function MonitorQuickStartGuide({monitor, orgId}: Props) {
```

Comment:
org slug as orgId is confusing
```suggestion
export default function MonitorQuickStartGuide({monitor, orgSlug}: Props) {
```